### PR TITLE
Release of Device Update for IoT Hub control-plane SDK library for .NET

### PR DESCRIPTION
**Pull request by dpokluda**
# All SDK Contribution checklist:

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] **Please open PR in `Draft` mode if it is:**
	- Work in progress or not intended to be merged.
	- Encountering multiple pipeline failures and working on fixes.
- [ ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/breaking-change-rules.md).**

### [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code. (Track 2 only)
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK. Please double check nuget.org current release version.

## Additional management plane SDK specific contribution checklist: 
Note: Only applies to `Microsoft.Azure.Management.[RP]` or `Azure.ResourceManager.[RP]`
 
- [x] Include updated [management metadata](https://github.com/Azure/azure-sdk-for-net/tree/master/eng/mgmt/mgmtmetadata).
- [x] Update AzureRP.props to add/remove version info to maintain up to date API versions.

### Management plane SDK Troubleshooting
- If this is very first SDK for a services and you are adding new service folders directly under /SDK, please add `new service` label and/or contact assigned reviewer.
- If the check fails at the `Verify Code Generation` step, please ensure:
	- Do not modify any code in generated folders.
	- Do not selectively include/remove generated files in the PR.
	- Do use `generate.ps1/cmd` to generate this PR instead of calling `autorest` directly.
	Please pay attention to the @microsoft.csharp version output after running generate.ps1. If it is lower than current released version (2.3.82), please run it again as it should pull down the latest version,

### Old outstanding PR cleanup
 Please note:
	If PRs (including draft) has been out for more than 60 days and there are no responses from our query or followups, they will be closed to maintain a concise list for our reviewers.

### CODE REVIEW

File: sdk/deviceupdate/Microsoft.Azure.Management.DeviceUpdate/src/Microsoft.Azure.Management.DeviceUpdate.csproj [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  <PropertyGroup>
+    <TargetFrameworks>$(SdkTargetFx)</TargetFrameworks>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.DeviceUpdate</PackageId>
+    <Description>Provides management capabilities for Microsoft Azure Device Update for IoT Hub.</Description>
+    <AssemblyTitle>Microsoft Azure Device Update for IoT Hub Management</AssemblyTitle>
+    <AssemblyName>Microsoft.Azure.Management.DeviceUpdate</AssemblyName>
+    <Version>1.0.0</Version>
```
Comment (m-nash) [inline sdk/deviceupdate/Microsoft.Azure.Management.DeviceUpdate/src/Microsoft.Azure.Management.DeviceUpdate.csproj:15]: I think this should be 1.0.0-beta.1

File: sdk/deviceupdate/Microsoft.Azure.Management.DeviceUpdate/tests/Microsoft.Azure.Management.DeviceUpdate.Tests.csproj [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  
+  <ItemGroup>
+    <Folder Include="SessionRecords\" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="FluentAssertions" Version="5.10.3" />
+    <PackageReference Include="Microsoft.Azure.Management.ResourceManager" Version="3.7.0-preview" />
+  </ItemGroup>
+</Project>
```
Comment (m-nash) [inline sdk/deviceupdate/Microsoft.Azure.Management.DeviceUpdate/tests/Microsoft.Azure.Management.DeviceUpdate.Tests.csproj:35]: nit: missing newline

File: sdk/deviceupdate/Microsoft.Azure.Management.DeviceUpdate/src/Properties/AssemblyInfo.cs [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+[assembly: AssemblyVersion("1.0.0.0")]
+[assembly: AssemblyFileVersion("1.0.0.0")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("Microsoft")]
+[assembly: AssemblyProduct("Microsoft Azure .NET SDK")]
+[assembly: AssemblyCopyright("Copyright (c) Microsoft Corporation")]
+[assembly: AssemblyTrademark("")]
+[assembly: AssemblyCulture("")]
+[assembly: NeutralResourcesLanguage("en")]
```
Comment (m-nash) [inline sdk/deviceupdate/Microsoft.Azure.Management.DeviceUpdate/src/Properties/AssemblyInfo.cs:18]: nit: missing newline

File: sdk/deviceupdate/Microsoft.Azure.Management.DeviceUpdate/AzSdk.RP.props [hunk 4]
```diff
@@ -0,0 +1,7 @@
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!--This file and it's contents are updated at build time moving or editing might result in build failure. Take due deligence while editing this file-->
+  <PropertyGroup>
+    <AzureApiTag>DeviceUpdate_2020-03-01;</AzureApiTag>
+    <PackageTags>$(PackageTags);$(CommonTags);$(AzureApiTag);</PackageTags>
+  </PropertyGroup>
+</Project>
```
Comment (m-nash) [inline sdk/deviceupdate/Microsoft.Azure.Management.DeviceUpdate/AzSdk.RP.props:7]: nit: missing newline

**Review by m-nash — DISMISSED** [review]

File: sdk/deviceupdate/Microsoft.Azure.Management.DeviceUpdate/src/Microsoft.Azure.Management.DeviceUpdate.csproj (hunk 1 above)
Comment (dpokluda) [reply sdk/deviceupdate/Microsoft.Azure.Management.DeviceUpdate/src/Microsoft.Azure.Management.DeviceUpdate.csproj:15]: Rigth, fixing it.

File: sdk/deviceupdate/Microsoft.Azure.Management.DeviceUpdate/Microsoft.Azure.Management.DeviceUpdate.sln [hunk 5]
```diff
@@ -0,0 +1,36 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.30114.128
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Azure.Management.DeviceUpdate", "src\Microsoft.Azure.Management.DeviceUpdate.csproj", "{2886F4B7-FEA8-42D5-BC8C-BDBCA917F957}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Azure.Management.DeviceUpdate.Tests", "tests\Microsoft.Azure.Management.DeviceUpdate.Tests.csproj", "{9F2DB215-83CD-48AE-BEC2-601E6EB50FE3}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution items", "Solution items", "{D1057B40-2779-4320-A41F-8380CDEB3400}"
```
Comment (m-nash) [inline sdk/deviceupdate/Microsoft.Azure.Management.DeviceUpdate/Microsoft.Azure.Management.DeviceUpdate.sln:10]: this shouldn't be needed as a solution item

**Review by allenjzhang — APPROVED** [review]

**Pull request merged**